package.json

```diff
@@ -24,10 +24,11 @@
     "packages/*"
   ],
   "engines": {
-    "node": "18.15.0",
+    "node": ">=18.16.0",
     "npm": ">=8"
   },
   "devDependencies": {
+    "azure-functions-core-tools": "^4.0.5455",
```

Comment:
I had to move the core tools at the root of the repo and regenerate the lockfile as `npm install` wasn't working anymore.
Moving this dependency to the root will avoid any version conflict during the time the `/api` and `api-v4` coexists.